include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1603,6 +1603,26 @@ FEPointEvaluation<n_components_, dim, spacedim, Number>::reinit(
   current_face_number = numbers::invalid_unsigned_int;
 
   do_reinit();
+
+  if (!fast_path)
+    {
+      const auto &triangulation = mapping_info->get_triangulation();
+      const auto  cell_level_and_index =
+        mapping_info->get_cell_level_and_index(cell_index);
+      const auto unit_points = mapping_info->get_unit_points_scalar(cell_index);
```

Comment:
I would prefer if we would spell out these types, because it makes the code much more readable.